Sniffo/Sniffo.CLI/CommandLineInterface.cs

```diff
@@ -16,6 +16,7 @@ public void Start()
         {
             Console.WriteLine("Sniffio: A network packets Sniffer");
             Console.WriteLine("Made By: TheLe0 - Leonardo Tosin");
+            Console.WriteLine("         aimperatori - Anderson Imperatori");
```

Comment:
Put on the line above, separated by comma. Depending on the terminal used, can break your identation.